tests/environments/storage/test_docker_storage.py

```diff
@@ -47,11 +47,13 @@ def test_empty_docker_storage(monkeypatch, platform, url):
     storage = Docker()
 
     assert not storage.registry_url
-    assert storage.base_image.startswith("python:")
+    assert storage.base_image.startswith("prefecthq/prefect:python")
     assert not storage.image_name
     assert not storage.image_tag
-    assert not storage.python_dependencies
-    assert not storage.env_vars
+    assert storage.python_dependencies == ["wheel"]
+    assert storage.env_vars == {
+        "PREFECT__USER_CONFIG_PATH": "/root/.prefect/config.toml"
```

Comment:
@joshmeek do you remember why we had to include this originally? I suspect it's no longer necessary but I don't want to take any chances

Reply:
I don't think this is used anymore

Reply:
don't remove it as part of this PR, but do remove it as part of another PR after this gets merged pls